prettier.config.js

```diff
@@ -4,8 +4,4 @@ module.exports = {
   singleQuote: true,
   tabWidth: 2,
   trailingComma: 'all',
-  plugins: [
-    // Automatically sorts tailwind classes to avoid specificity issues
-    'prettier-plugin-tailwindcss', // MUST COME LAST
```

Comment:
Why is this being removed?

Reply:
This is misconfigured and not working. I'll need you to look at this later to get it working.

Reply:
Will do!